bot/exts/utils/snekbox.py

```diff
@@ -248,18 +319,25 @@ async def continue_eval(self, ctx: Context, response: Message) -> Optional[str]:
                     timeout=10
                 )
 
-                code = await self.get_code(new_message)
+                code = await self.get_code(new_message, ctx.command)
```

Comment:
While going through this I noticed we don't check for None on the code result which can cause an error down the line